lib/agent.js

```diff
@@ -281,8 +306,24 @@ Agent.prototype.captureError = function (err, opts, cb) {
   }
 
   function send (error) {
-    agent.logger.info('logging error %s with Elastic APM', id)
-    request.errors(agent, [error], cb)
+    error = agent._filters.process(error) // TODO: Update filter to expect this format
+
+    if (!error) {
+      agent.logger.debug('error ignored by filter %o', {id: id})
+      cb()
+      return
+    }
+
+    truncate.error(error, agent._conf)
+
+    agent.logger.info(`Sending error ${id} to Elastic APM`)
+    if (agent._apmServer) {
+      agent._apmServer.sendError(error, function () {
+        agent._apmServer.flush(cb)
+      })
+    } else {
+      process.nextTick(cb)
```

Comment:
Can you log something here so we can tell when someone tries to send something without `_apmServer` set?

Reply:
fixed